src/components/Header-Footer/Header.vue

```diff
@@ -1,8 +1,18 @@
 <template>
-  <div class="login">
-    <h1>{{ msg }}</h1>
-    <button v-on:click="greet" class="enableEthereumButton">Enable Ethereum</button>
-  </div>
+<div class="fluid-container sticky-top">
+<b-navbar variant="faded" type="light">
+    <b-navbar-brand class="ml-auto" href="#">
+      <img :src="require('../../assets/CATS.png')" alt="" width="140" height="80"/>
+      <a font-weight= "900"> C.A.T.S</a>
+    </b-navbar-brand>
+    
+    <b-navbar-button class="col"> 
+         <li>{{ msg }}</li>
+         <button v-on:click="greet" class="enableEthereumButton">Connect</button>
```

Comment:
At some point we'll probably want to pull this button (and the greet function) out into its own component. For now we can leave it like you have it.